.github/workflows/modmesh_viewer.yml

```diff
@@ -15,8 +15,9 @@ jobs:
 
     strategy:
         matrix:
-          os: [ubuntu-22.04, macos-12]
+          os: [ubuntu-22.04, macos-12, windows-2022]
           cmake_build_type: [Release, Debug]
+          arch: [amd64]
```

Comment:
Do we need it?  Isn't the cloud runner always uses x86 (https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)?

Reply:
Hmm, I don't really know about this. It's the input to msvc-dev-cmd and the default value seems sufficient, I think we could just delete the line.

Reply:
Agree.  If it's not needed, let's delete it.